diff --git a/extensions/pref/autoconfig/src/nsReadConfig.cpp b/extensions/pref/autoconfig/src/nsReadConfig.cpp
index 06f54dd0ab77..54f08ac90104 100644
--- a/extensions/pref/autoconfig/src/nsReadConfig.cpp
+++ b/extensions/pref/autoconfig/src/nsReadConfig.cpp
@@ -247,16 +247,8 @@ nsresult nsReadConfig::openAndEvaluateJSFile(const char* aFileName,
   nsCOMPtr<nsIInputStream> inStr;
   if (isBinDir) {
     nsCOMPtr<nsIFile> jsFile;
-#if defined(MOZ_WIDGET_GTK)
-    if (!mozilla::widget::IsRunningUnderFlatpakOrSnap()) {
-#endif  // defined(MOZ_WIDGET_GTK)
-      rv = NS_GetSpecialDirectory(NS_GRE_DIR, getter_AddRefs(jsFile));
-#if defined(MOZ_WIDGET_GTK)
-    } else {
-      rv = NS_GetSpecialDirectory(NS_OS_SYSTEM_CONFIG_DIR,
-                                  getter_AddRefs(jsFile));
-    }
-#endif  // defined(MOZ_WIDGET_GTK)
+    rv = NS_GetSpecialDirectory(NS_GRE_DIR, getter_AddRefs(jsFile));
+
     if (NS_FAILED(rv)) return rv;
 
     rv = jsFile->AppendNative(nsDependentCString(aFileName));
