diff --git a/browser/extensions/moz.build b/browser/extensions/moz.build
index 4c9fa78..095d199 100644
--- a/browser/extensions/moz.build
+++ b/browser/extensions/moz.build
@@ -5,10 +5,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 DIRS += [
-    'doh-rollout',
     'formautofill',
     'pdfjs',
-    'screenshots',
-    'webcompat',
-    'report-site-issue'
+    'screenshots'
 ]
