diff --git a/build/moz.configure/toolchain.configure b/build/moz.configure/toolchain.configure
index 627596b..6c3b5b3 100755
--- a/build/moz.configure/toolchain.configure
+++ b/build/moz.configure/toolchain.configure
@@ -1433,6 +1433,9 @@ set_config('_DEPEND_CFLAGS', depend_cflags(c_compiler))
 set_config('_HOST_DEPEND_CFLAGS', depend_cflags(host_c_compiler))
 
 
+add_old_configure_assignment('CXXFLAGS', ['--gcc-toolchain=/usr/lib/gcc-mozilla'])
+
+
 @depends(c_compiler)
 def preprocess_option(compiler):
     # The uses of PREPROCESS_OPTION depend on the spacing for -o/-Fi.
@@ -2116,7 +2119,7 @@ def linker_ldflags(linker, macos_sdk):
             flags.append('-Wl,-syslibroot,%s' % macos_sdk)
         else:
             flags.append('-Wl,--sysroot=%s' % macos_sdk)
-
+    flags.append('-static-libstdc++')
     return flags
 
 
diff --git a/servo/components/style/build_gecko.rs b/servo/components/style/build_gecko.rs
index a502b7d..28370f6 100644
--- a/servo/components/style/build_gecko.rs
+++ b/servo/components/style/build_gecko.rs
@@ -148,6 +148,8 @@ impl BuilderExt for Builder {
             builder = builder.with_rustfmt(path);
         }
 
+        builder = builder.clang_arg("--gcc-toolchain=/usr/lib/gcc-mozilla");
+
         for dir in SEARCH_PATHS.iter() {
             builder = builder.clang_arg("-I").clang_arg(dir.to_str().unwrap());
         }
