diff -r 246cfe5d5a05 build/moz.configure/rust.configure
--- a/build/moz.configure/rust.configure	Sat Oct 02 16:40:36 2021 +0000
+++ b/build/moz.configure/rust.configure	Sun Oct 03 11:00:29 2021 +0200
@@ -171,7 +171,7 @@
             )
         )
     rustc_min_version = Version(MINIMUM_RUST_VERSION)
-    cargo_min_version = rustc_min_version
+    cargo_min_version = Version("1.51.0")
 
     version = rustc_info.version
     is_nightly = "nightly" in version.version
